.formatter.exs

```diff
@@ -27,5 +27,7 @@ locals_without_parens = [
   locals_without_parens: locals_without_parens,
   export: [
     locals_without_parens: locals_without_parens
-  ]
+  ],
+  line_length: 210,
+  inputs: ["*.{ex,exs}", "{config,lib,test}/**/*.{ex,exs}"]
```

Comment:
This should not be a part of this `PR` …

Reply:
Will be removed before merging